library/src/utils.cxx

```diff
@@ -25,7 +25,7 @@ unsigned int utils::textDistance(std::string_view strA, std::string_view strB)
 }
 
 //----------------------------------------------------------------------------
-std::vector<std::string> utils::tokenize(std::string_view str)
+std::vector<std::string> utils::tokenize(std::string_view str, bool skipComment)
```

Comment:
so `skipComment` skips the skipping of comments?
Should it be `keepComments` or `enableComments` instead?